net-misc/openntpd/openntpd-6.2_p3-r1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Old wrongly typed copyright header. Use repoman to commit so it fixes this automatically.